lib/kafka_ex/server.ex

```diff
@@ -15,14 +15,22 @@ defmodule KafkaEx.Server do
     {:ok, {metadata, client, nil}}
   end
 
-  defp send_request(metadata, client, topic, partition, request_fn, parser, timeout \\ 100) do
+  defp send_request(metadata, client, topic, partition, request_fn, parser, timeout \\ 100, return_response \\ true) do
```

Comment:
This function always has a return value,  `return_response` has added an extra arg plus extra lines of code, the only case where we cal `send_request` without the need for a response already _devnulls_ the response.